### ISSUE TITLE
bundleGraph.isAssetReachableFromBundle is not a function when using TypeScript

### ISSUE THREAD

**Issue by pojntfx** [parcel-bundler/parcel#4574]
It doesn't seem to be possible to use TypeScript with Parcel V2:

```json
"@parcel/config-default": "nightly",
"@parcel/transformer-typescript-tsc": "nightly",
"parcel": "nightly",
```

```json
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{ts,tsx}": ["@parcel/transformer-typescript-tsc"]
  }
}
```

```json
{
  "presets": [
    "@babel/preset-env",
    "@babel/preset-react"
  ]
}
```

This fails with the following error:

```bash
bundleGraph.isAssetReachableFromBundle is not a function when using TypeScript
```

**Comment by astegmaier** [comment]
When I tried out your config, I saw that parcel was auto-installing some packages that were not included in your package.json, but it chose to fix the version of these new packages to `2.0.0-alpha.3.2`. Quite a bit has happened between the latest nightly packages (currently `2.0.0-nightly.250`ish) and alpha.3.2, so I wouldn't expect an installation with mixed versions to work. Can you make sure that all packages that get installed are fixed to the latest nighly build? If you want to debug further, could you share a link to a sample repo?

(Also, it's not required to use `@parcel/transformer-typescript-tsc` unless you want to use `tsc` instead of `babel` to do the transpilation. While you're getting started, it might make sense to get a simple `babel`-based config working, and then go from there.)

**Comment by pojntfx** [comment]
@astegmaier Thank you so much for your support! I'm just using babel now, which works just fine; after adding a `tsconfig.json` and `pnpify` (I'm using Yarn V2) even VSCode uses the correct TS version.
